.prettierrc.json

```diff
@@ -0,0 +1,8 @@
+{
+  "tabWidth": 2,
+  "semi": false,
+  "singleQuote": true,
+  "trailingComma": "es5",
+  "printWidth": 100,
```

Comment:
Did you find 100 width would wrap stuff more than you wanted? This project has a lot of long lines because of CSS variable names so I wonder if a higher printWidth is better. (granted, larger printWidths make split screens more difficult)

I don't have an opinion either way, just something to consider.

Reply:
I could probably bump it to 110 or 120 and be happy with it.